example/withSubscription.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+import React from 'react';
+import { render } from 'react-dom';
+import {
+  Provider,
+  Subscribe,
+  Container,
+  withSubscription
+} from '../src/unstated';
+
+type CounterState = {
+  count: number
+};
+
+class CounterContainer extends Container<CounterState> {
+  state = { count: 0 };
+
+  increment() {
+    this.setState({ count: this.state.count + 1 });
+  }
+
+  decrement() {
+    this.setState({ count: this.state.count - 1 });
+  }
+}
+
+const Counter = withSubscription({ counter: CounterContainer })(
```

Comment:
it would be cool if the example could show that you can use those props in something else than render